service/src/test/java/greencity/service/HabitAssignServiceImplTest.java

```diff
@@ -287,12 +289,36 @@ void unenrollHabitThrowWrongIdException() {
     }
 
     @Test
-    void deleteHabitAssign() {
+    void changeStatusHabitAssign() {
         HabitAssign habitAssign = ModelUtils.getHabitAssign();
-        when(habitAssignRepo.findByUserIdAndHabitId(1L, 1L)).thenReturn(Optional.ofNullable(habitAssign));
-        assert habitAssign != null;
-        habitAssignService.deleteHabitAssign(1L, 1L);
-        verify(habitAssignRepo).delete(habitAssign);
+        habitAssign.getHabit().setHabitAssigns(generateHabitAssignList());
+        when(habitAssignRepo.findListByUserIdAndHabitIdAndStatusIsInProgress(1L, 1L))
+            .thenReturn(habitAssign.getHabit().getHabitAssigns());
+        habitAssignService.cancelAllHabitAssign(1L, 1L);
+        Mockito.verify(userShoppingListItemRepo).deleteByShoppingListItemsByHabitAssignId(1L);
```

Comment:
Please add verify for `userShoppingListItemRepo).deleteByShoppingListItemsByHabitAssignId(2L)`